LEAF_Nexus/sources/System.php

```diff
@@ -242,4 +256,90 @@ public function refreshOrgchartEmployees()
 
         return $ret;
     }
+
+    /**
+     * Get primary admin.
+     *
+     * @return array|false array with primary admin's info
+     */
+    public function getPrimaryAdmin()
+    {
+        $strSQL = "SELECT `data` FROM `settings` WHERE `setting` = 'primaryAdmin'";
+        $primaryAdminRes = $this->db->query($strSQL);
+
+        include_once 'Employee.php';
+        $employee = new Orgchart\Employee($this->db, $this->login);
+
+        if(count($primaryAdminRes))
```

Comment:
This needs to a comparison.  if(count($primaryAdminRes) > 0)

Reply:
Added comparison